### PR TITLE
add  test environment conf

### PR DESCRIPTION
**Pull request by xwz123**

### CODE REVIEW

File: applications/jenkins-test/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                  key: jenkins-admin-password
+                  name: openlookeng-jenkins
+            - name: ADMIN_USER
+              valueFrom:
+                secretKeyRef:
+                  key: jenkins-admin-user
+                  name: openlookeng-jenkins
+          resources:
+            requests:
+              cpu: 8000m
```
Comment (suisuisuixin) [inline applications/jenkins-test/deployment.yaml:24]: suggest 2000m

File: applications/jenkins-test/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+              memory: 16000Mi
```
Comment (suisuisuixin) [inline applications/jenkins-test/deployment.yaml:25]: suggest 2000Mi

File: applications/jenkins-test/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                  name: openlookeng-jenkins
+            - name: ADMIN_USER
+              valueFrom:
+                secretKeyRef:
+                  key: jenkins-admin-user
+                  name: openlookeng-jenkins
+      volumes:
+        - name: jenkins-home
+          persistentVolumeClaim:
+            claimName: cce-evs-jenkins-master-data
```
Comment (suisuisuixin) [inline applications/jenkins-test/deployment.yaml:45]: need the same as pvc.metadata.name

File: applications/jenkins-test/pvc.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: jenkins-test
```
Comment (suisuisuixin) [inline applications/jenkins-test/pvc.yaml:5]: jenkins

File: applications/jenkins-test/deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: jenkins-test
```
Comment (suisuisuixin) [inline applications/jenkins-test/deployment.yaml:5]: jenkins

File: applications/jenkins-test/deployment.yaml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+spec:
+  template:
+    spec:
+      containers:
+        - name: jenkins-test
```
Comment (suisuisuixin) [inline applications/jenkins-test/deployment.yaml:10]: jenkins

File: applications/jenkins-test/kustomization.yaml [hunk 7]
```diff
@@ -0,0 +1,11 @@
+resources:
+  - https://github.com/opensourceways/app-jenkins/deploy-without-agent?ref=stable
+  - secrets.yaml
+  - namespace.yaml
```
Comment (suisuisuixin) [inline applications/jenkins-test/kustomization.yaml:4]: need add - ingress.yaml

**Review by TommyLike — APPROVED** [review]

**Pull request merged**